TLM/TLM/TLM.csproj

```diff
@@ -13,6 +13,7 @@
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
     <LangVersion>preview</LangVersion>
+    <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
```

Comment:
why?

Reply:
@krzychu124 I do need `TrafficManager.Constants` so I need to have a reference to TMPE dll. But that dll is big with all the pictures and all. With this, I can copy a smaller reference assembly to my repo for reference.

Reply:
4MB is big... whatever.